File: samples/lambda-layers/javascript/src/handler.js
/**
 * Lambda function that uses the shared layer.
 * The layer is extracted to /opt/nodejs/ at runtime.
 */

const { echo } = require('/opt/nodejs/lib');

module.exports.hello = async function(event, context) {
  const message = 'Hello from Lambda Layer!';
  echo(message);

  return {
    statusCode: 200,
    body: JSON.stringify({
      message: message,
      layerWorking: true
    })
  };
}

File: samples/lambda-container-image/python/handler.py
"""Lambda handler for container image demo."""

import json


def handler(event, context):
    """Simple handler that echoes the event."""
    print("Hello from LocalStack Lambda container image!")

    return {
        "statusCode": 200,
        "body": json.dumps({
            "message": "Hello from Lambda container image!",
            "event": event
        })
    }
